### PR TITLE
Slides on Admins / Empathy

### PR DESCRIPTION
**Pull request by erasche**

### CODE REVIEW

File: topics/admin/tutorials/empathy/slides.html [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+We are not just janitors / code monkeys
+- We're humans with feelings
+- Our users are humans with feelings as well
+- Both groups need to be understanding of each other
+
+We're (often) the point of contact for any problems
+- It's our responsibilty to fix Galaxy problems
+- Or any server problems for our usres
+- Of sometimes anything that is electrical is our problem
```
Comment (Slugger70) [inline topics/admin/tutorials/empathy/slides.html]: ?? Sometimes things that aren't meant to be our problem become ours ??

Comment (Slugger70) [reply topics/admin/tutorials/empathy/slides.html]: Or is this about Electrical supply?

File: topics/admin/tutorials/empathy/slides.html [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+## Part of the Job
+
+We are not just janitors / code monkeys
+- We're humans with feelings
+- Our users are humans with feelings as well
+- Both groups need to be understanding of each other
+
+We're (often) the point of contact for any problems
+- It's our responsibilty to fix Galaxy problems
+- Or any server problems for our usres
```
[the remark this answers is not in the document]
Comment (erasche) [reply topics/admin/tutorials/empathy/slides.html]:
```suggestion
- Or any server problems for our users
```

File: topics/admin/tutorials/empathy/slides.html (hunk 1 above)
Comment (erasche) [reply topics/admin/tutorials/empathy/slides.html]:
```suggestion
- Or sometimes anything that is electrical is our problem
```

File: topics/admin/tutorials/empathy/slides.html [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+???
+
+We need to talk and not just talk but communicate with each other, listen to each other's complaints and problems. You can only control 1/2 of the equation, but you can have a positive impact on the balance.
+
+You're also often the Tier 1 response to issues, so even when you're stressed, users often just want to feel heard, like they've told someone and their complaint has been heard. Doing this can help defuse bad situations where poeple are upset and feelings are hurt.
+
+---
+
+## SLAs vs SLOs
```
Comment (nsoranzo) [inline topics/admin/tutorials/empathy/slides.html]: Maybe explain the acronyms?

File: topics/admin/tutorials/empathy/slides.html [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+	- [Write posts after unplanned outages](https://galaxyproject.eu/posts/2018/09/21/downtime/)
+	- [Add graphics](https://galaxyproject.eu/posts/2018/07/12/workflow-outage/)
+- Try and be empathetic in your communications
+	- Give users the benefit of the doubt
+	- Acknowledge their feelings but try not to take it as a personal attack (even when it is.)
+- For any message you write, imagine you are reading it sent to you. How would it make you feel?
+
+???
+
+common advice: Respond calmly, do *not* send emails immediately, come back to it in a few hours
```
Comment (nsoranzo) [inline topics/admin/tutorials/empathy/slides.html]:
```suggestion
Common advice: respond calmly, do *not* send emails immediately, come back to it in a few hours
```

File: topics/admin/tutorials/empathy/slides.html [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+
+---
+
+## Shared Responsibility
+
+Documentation *Documentation* **Documentation**
+
+- Document your code/infrastructure/everything
+	- [Various types of documentation](https://en.wikipedia.org/wiki/Software_documentation)
+- You will probably change jobs, and someone else will maintain your code
```
Comment (nsoranzo) [inline topics/admin/tutorials/empathy/slides.html]:
```suggestion
- You will probably change jobs, and someone else will have to maintain your code
```

File: topics/admin/tutorials/empathy/slides.html [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+Explicit Goals:
+
+--
+
+1. You're happy
+2. Your users are happy
+
+--
+
+It is not a zero sum game, both of these can be true (or true *enough*), and both of these are worth working towards.
```
Comment (nsoranzo) [inline topics/admin/tutorials/empathy/slides.html]:
```suggestion
It is not a zero-sum game, both of these can be true (or true *enough*), and both of these are worth working towards.
```

File: topics/admin/tutorials/empathy/slides.html [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+------- | ---   | ------------           | -----    | ------------
+Haproxy | 99.9% | 43 minutes             | Yes      | Yes
+Cluster | 95.0% | 36 hours               | Yes      | No
+Sentry  | 50.0% | 15 day                 | No       | No
+Jenkins | 50.0% | 15 day                 | No       | Yes-ish
+Grafana | 50.0% | 15 day                 | No       | Yes-ish
+
+???
+
+Service Level Ojectives are just your goals, not a legal agreement with your users. It's just a number that you plan to hit, that you can share with your users to give them an idea of your goals.
```
Comment (nsoranzo) [inline topics/admin/tutorials/empathy/slides.html]:
```suggestion
Service Level Objectives are just your goals, not a legal agreement with your users. It's just a number that you plan to hit, that you can share with your users to give them an idea of your goals.
```

**Pull request merged**